server/svix-server/tests/utils/common_calls.rs

```diff
@@ -181,7 +181,7 @@ pub async fn common_test_list<
     assert!(!list.done);
 
     let list = client
-        .get::<ListResponse<ModelOut>>(&format!("{}?limit=500", path), StatusCode::OK)
+        .get::<ListResponse<ModelOut>>(&format!("{}?limit=50", path), StatusCode::OK)
```

Comment:
Everything passed for me, it was just checking what happens if the limit was over the count I think. Regardless it errors if you keep it at 500 because it returns 422 now

Reply:
Yeah, it should return 422, that's why I'm surprised this works.